server/server.js

```diff
@@ -35,6 +36,10 @@ app.get('/trees', function (req, res) {
     join = ", certificates, donors, organizations";
     joinCriteria = "AND trees.certificate_id = certificates.id AND certificates.donor_id = donors.id AND donors.organization_id = organizations.id AND organizations.id = " + organization;
     subset = true;
+  } else if (flavor) {
+    join = "INNER JOIN tree_attributes ON tree_attributes.tree_id = trees.id";
+    joinCriteria = "AND tree_attributes.key = 'app_flavor' AND tree_attributes.value = '" + flavor + "'";
```

Comment:
Could use a template string, but none of the other code does so it's probably not a big deal

Reply:
i find template strings atractive, but i am married to consistency :)